--- origsrc/inkscape-0.91/src/live_effects/parameter/powerstrokepointarray.cpp	2014-11-30 12:45:32.000000000 -0600
+++ src/inkscape-0.91/src/live_effects/parameter/powerstrokepointarray.cpp	2015-04-21 21:41:05.097481600 -0500
@@ -183,7 +183,7 @@ PowerStrokePointArrayParamKnotHolderEnti
     using namespace Geom;
 
     if (!valid_index(_index)) {
-        return Geom::Point(infinity(), infinity());
+        return Geom::Point(Geom::infinity(), Geom::infinity());
     }
 
     Piecewise<D2<SBasis> > const & pwd2 = _pparam->get_pwd2();
@@ -192,7 +192,7 @@ PowerStrokePointArrayParamKnotHolderEnti
     Point offset_point = _pparam->_vector.at(_index);
     if (offset_point[X] > pwd2.size() || offset_point[X] < 0) {
         g_warning("Broken powerstroke point at %f, I won't try to add that", offset_point[X]);
-        return Geom::Point(infinity(), infinity());
+        return Geom::Point(Geom::infinity(), Geom::infinity());
     }
     Point canvas_point = pwd2.valueAt(offset_point[X]) + offset_point[Y] * n.valueAt(offset_point[X]);
     return canvas_point;
