--- origsrc/inkscape-e86c870879/src/path-prefix.cpp	2021-01-16 00:43:56.000000000 +0900
+++ src/inkscape-e86c870879/src/path-prefix.cpp
@@ -109,7 +109,7 @@
         } else {
             g_warning("get_program_name() - _NSGetExecutablePath failed");
         }
-#elif defined(__linux__)
+#elif defined(__linux__) || defined(__CYGWIN__)
         program_name = g_file_read_link("/proc/self/exe", NULL);
         if (!program_name) {
             g_warning("get_program_name() - g_file_read_link failed");
