--- origsrc/inkscape-0.46/configure.ac	2008-03-10 23:47:01.000000000 -0500
+++ src/inkscape-0.46/configure.ac	2008-06-16 18:12:41.250000000 -0500
@@ -561,18 +561,16 @@
 dnl Check for ImageMagick Magick++ 
 dnl ******************************
 
-AC_CHECK_LIB(Magick++, InitializeMagick, [AC_CHECK_HEADER(Magick++.h, magick_ok=yes, magick_ok=no)], magick_ok=no, -llcms -lfreetype -ljpeg -lfontconfig -lXext -lSM -lICE -lX11 -lz -lpthread -lm -lpthread)
-
-if test "x$magick_ok" = "xyes"; then
-	AC_CHECK_PROG(MAGICKPP_CONFIG, Magick++-config, yes, no)
-	if test "x$MAGICKPP_CONFIG" = "xyes"; then
-		IMAGEMAGICK_LIBS=`Magick++-config --libs`
-		AC_SUBST(IMAGEMAGICK_LIBS)
-		AC_DEFINE(WITH_IMAGE_MAGICK,1,[Image Magick++ support for bitmap effects])
-	fi
+AC_CHECK_PROG(MAGICKPP_CONFIG, Magick++-config, yes, no)
+if test "x$MAGICKPP_CONFIG" = "xyes"; then
+	IMAGEMAGICK_CPPFLAGS="`Magick++-config --cppflags`"
+	IMAGEMAGICK_LIBS="`Magick++-config --libs`"
+	AC_SUBST(IMAGEMAGICK_CPPFLAGS)
+	AC_SUBST(IMAGEMAGICK_LIBS)
+	AC_DEFINE(WITH_IMAGE_MAGICK,1,[Image Magick++ support for bitmap effects])
 fi
 
-AM_CONDITIONAL(USE_IMAGE_MAGICK, test "x$magick_ok" = "xyes")
+AM_CONDITIONAL(USE_IMAGE_MAGICK, test "x$MAGICKPP_CONFIG" = "xyes")
 
 dnl ******************************
 dnl   Unconditional dependencies
--- origsrc/inkscape-0.46/src/Makefile.am	2008-03-10 23:44:31.000000000 -0500
+++ src/inkscape-0.46/src/Makefile.am	2008-06-16 18:12:41.265625000 -0500
@@ -14,17 +14,19 @@
 AUTOMAKE_OPTIONS = 1.7 subdir-objects
 
 INCLUDES =	\
-	$(PERL_CFLAGS) $(PYTHON_CFLAGS)	\
 	$(FREETYPE_CFLAGS)	\
 	$(GNOME_PRINT_CFLAGS)	\
 	$(GNOME_VFS_CFLAGS)	\
 	$(INKBOARD_CFLAGS) \
+	$(IMAGEMAGICK_CPPFLAGS) \
 	$(LIBWPG_CFLAGS) \
 	$(XFT_CFLAGS)	\
 	$(POPPLER_CFLAGS)	\
 	$(POPPLER_GLIB_CFLAGS)	\
 	-DPOTRACE=\"potrace\"	\
 	$(INKSCAPE_CFLAGS) \
+	$(PERL_CFLAGS) $(PYTHON_CFLAGS)	\
+	-I$(srcdir)/extension/script \
 	-I$(top_srcdir)/cxxtest
 
 include Makefile_insert
--- origsrc/inkscape-0.46/src/util/ucompose.hpp	2008-03-10 23:20:47.000000000 -0500
+++ src/inkscape-0.46/src/util/ucompose.hpp	2008-06-16 18:12:41.296875000 -0500
@@ -44,6 +44,10 @@
 #include <list>
 #include <map>			// for multimap
 
+#if defined(__CYGWIN__) && !defined(_GLIBCXX_USE_WSTRING)
+namespace std { typedef basic_string<wchar_t> wstring; }
+#endif
+
 namespace UStringPrivate
 {
   // the actual composition class - using String::ucompose is cleaner, so we
--- origsrc/inkscape-0.46/utf8-to-roff	2008-03-10 23:20:51.000000000 -0500
+++ src/inkscape-0.46/utf8-to-roff	2008-06-16 18:12:41.328125000 -0500
@@ -1,4 +1,4 @@
-#! /usr/bin/perl -w -CI
+#! /usr/bin/perl -w
 # Converts a man page from utf8 (not understood by groff 1.18) to groff escapes.
 # I couldn't find any existing tool to convert from utf8 to groff, though I
 # seem to recall seeing some code to do so somewhere.
